### PR TITLE
[rbs] fix tiny tweak 24287

### LINKED ISSUE #24287: Tiny change I forgot when implemented fix for the sso login button style
### Describe the Improvement

I forgot to fix the sso login icon box margin. Tiny cosmetic change related with https://github.com/directus/directus/issues/24118
I will submit a new PR for this one.

### PR DESCRIPTION
**Pull request by robsoncombr**
Before the change we have margin bigger than the line I noticed it before but during some changes it was lost from the previous PR so I'm sending this one as a re-opening action.

I made the red background to make it easier to note the difference it tiny but the right is really 1px;

With margin 2px;

image

With margin 1px;

image

Fixes #24287 24287

### CODE REVIEW

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: api/src/auth/drivers/oauth2.ts [hunk 1]
```diff
@@ -77,8 +77,20 @@ export class OAuth2AuthDriver extends LocalAuthDriver {
...
+
+		Causes the error:
+			throw new TypeError('client_id is required');
+
+		To be reverted and fixed by the implemented permanent solution.
+		*/
 		this.client = new issuer.Client({
-			client_id: clientId,
+			// client_id: clientId,
+			client_id: clientId.toString(),
```
[the remark this answers is not in the document]
Comment (robsoncombr) [reply api/src/auth/drivers/oauth2.ts:93]: Oops 🤭

**Comment by paescuj** [comment]
@robsoncombr Thanks for this pull request! ❤️ 

Could you help me understand why you want to reduce the margin / border-width to 1px?
The current value of 2px is in line with e.g. the border with of input fields and 1px is hardly visible anymore in my opinion.

I don't see an issue with the current styling:

login-page

**Comment by robsoncombr** [comment]
@paescuj interesting, and here is different for the record, windows 11 + edge
Allow me to review this, will let you know..

**Comment by robsoncombr** [comment]
@paescuj you know what? you are right while I'm old and needing to change my glasses.

took new screenshots with max zoom with original code.

no hover
image

hover
image
image

@rijkvanzanten ouch!

better we forget this all and close as being a false alarm 😂 it was just a crazy blind programmer.

and happy new year to everyone!

**Comment by AlexGaillard** [comment]
No worries @robsoncombr. Thanks for all your hard work. 👍 And HNY to you too 🎉

**Pull request closed without merging**